packages/adminrouter/extra/dcos-adminrouter.service

```diff
@@ -23,6 +23,6 @@ ExecStartPre=/usr/bin/test -f /var/lib/dcos/cluster-id
 # Wait for auth backend to seed the secret key
 ExecStartPre=/usr/bin/curl --fail -sS -o /dev/null 127.0.0.1:8101/acs/api/v1/groups
 ExecStart=$PKG_PATH/nginx/sbin/nginx -c $PKG_PATH/nginx/conf/nginx.master.conf
-ExecReload=/usr/bin/kill -HUP $MAINPID
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
Two things:
1. We need to confirm that `/bin/kill` exists on all the distros we currently deploy against (CoreOS, Ubuntu, CentOS)... I think adding a simple test for it's existence in our standard test suite would be sufficient. I would be surprised if it's not... but I've been surprised in this area before.
2. Systemd [ExecReload docs](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecReload=) strongly recommends to wait synchronously for the reload to complete and not just rely on `kill -HUP`.